README.md

```diff
@@ -12,10 +12,16 @@ Basically we use this config:
 ```
 vpcDiscovery:
     vpcName: '${opt:env}'
-    subnetNames: # optional if securityGroupNames are specified
-      - '${opt:env}_<name of subnet>'
-    securityGroupNames: # optional if subnetNames are specified
-      - '${opt:env}_<name of security group>'
+    subnets: # optional if `securityGroups` option is specified
```

Comment:
The line above probably needs to change since it's not only by name anymore `The vpc discovery plugin takes the given vpc, subnet, and security group names in the serverless file to setup the vpc configuration for the lambda.`